### ISSUE TITLE
Variable only used in format string reported as "is not accessed"

### ISSUE THREAD

**Issue by CTimmerman** [microsoft/pyright#10034]
## Environment data

-   Pylance version: 2025.2.1
-   OS and version: Windows 11 23H2
-   Python version (& distribution if applicable, e.g. Anaconda): 3.10.11

## Code Snippet

```python
escaped_command = "foo"
print(rf"\#0\f7> {escaped_command}")
```

## Repro Steps

1. Use Pylance and notice the faded variable and false report of variable not being accessed while it is via format string.

## Expected behavior

Notice format string variable use.

## Actual behavior

![Image](https://github.com/user-attachments/assets/48d60ade-fd76-4b42-bc2b-fd0f07d84237)

XXX

## Logs

```
XXX
```

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.397.
